### PR TITLE
Fix issues of attention.core_attention.softmax_offset is None

### PR DESCRIPTION
**Pull request by yueshen2016**
## What does this PR do?

**Type of change:** ? 
Bug fix
**Overview:** ?
Fix issues of attention.core_attention.softmax_offset is None for megatron import
## Usage

```python
# Add a code snippet demonstrating how to use this
```

## Before your PR is "*Ready for review*"

- **Make sure you read and follow [Contributor guidelines](https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/main/CONTRIBUTING.md)** and your commits are signed.
- **Is this change backward compatible?**: Yes/No 
- **Did you write any new necessary tests?**: Yes/No
- **Did you add or update any necessary documentation?**: Yes/No
- **Did you update [Changelog](https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/main/CHANGELOG.rst)?**: Yes/No 

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved robustness when importing/exporting models by safely handling missing or undefined softmax offsets.
  * Prevents errors or crashes in certain attention configurations where an offset was absent.
  * No changes to public APIs; behavior unchanged when a valid offset is provided.
  * Enhances stability and reliability for affected import/export workflows.

### CODE REVIEW

File: modelopt/torch/export/plugins/megatron_importer.py [hunk 1]
```diff
@@ -512,7 +512,10 @@ def _import_state_dict(self):
                             self.rules["k_layernorm"](attention.k_layernorm, layer_id)
                         self.rules["linear_qkv"](attention.linear_qkv, layer_id)
                         self.rules["linear_proj"](attention.linear_proj, layer_id)
-                        if hasattr(attention.core_attention, "softmax_offset"):
+                        if (
```
Comment (ChenhanYu) [inline modelopt/torch/export/plugins/megatron_importer.py]:
```
if getattr(attention.core_attention, "softmax_offset", None) is not None:
```
